apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -40,13 +40,12 @@ const DATE_PARSING_FORMATS = [
 const READINGS_LIMIT = 500
 
 const ERRORS = {
-    TOO_MUCH_READINGS: (sentCount) => ({
+    TOO_MUCH_READINGS: {
```

Comment:
```suggestion
    TOO_MANY_READINGS: {
```

Reply:
too much to do it in all places in this pr